### PR TITLE
Basic Template to create VTEX Plugins

### PR DESCRIPTION
**Pull request by VerasThiago**
#### What is the purpose of this pull request?
Basic template that the developer can use to create his own VTEX Plugins to `toolbelt`

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: oclif.manifest.json [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+{"version":"0.0.0","commands":{"hello":{"id":"hello","description":"describe the command here","pluginName":"cli-plugin-template","pluginType":"core","aliases":[],"examples":["$ oclif-example hello\nhello world from ./src/hello.ts!\n"],"flags":{"verbose":{"name":"verbose","type":"boolean","char":"v","description":"Show debug level logs","allowNo":false},"help":{"name":"help","type":"boolean","char":"h","description":"show CLI help","allowNo":false},"trace":{"name":"trace","type":"boolean","description":"Ensure all requests to VTEX IO are traced","allowNo":false},"name":{"name":"name","type":"option","char":"n","description":"name to print"},"force":{"name":"force","type":"boolean","char":"f","allowNo":false}},"args":[{"name":"file"}]}}}
```
Comment (tiagonapoli) [inline oclif.manifest.json:1]: This file can be deleted

File: tsconfig.tsbuildinfo [hunk 2]
```diff
@@ -26,6 +26,21 @@
         "signature": "5f4e733ced4e129482ae2186aae29fde948ab7182844c3a5a51dd346182c7b06",
         "affectsGlobalScope": false
       },
+      "./node_modules/typescript/lib/lib.es2019.d.ts": {
```
Comment (tiagonapoli) [inline tsconfig.tsbuildinfo:29]: This file can be deleted

File: .gitignore [hunk 3]
```diff
@@ -9,4 +9,5 @@
 /tmp
 coverage
 node_modules
-package-lock.json
\ No newline at end of file
+oclif.manifest.json
+tsconfig.tsbuildinfo
```
Comment (tiagonapoli) [inline .gitignore:13]: Add `.eslintcache`

File: tsconfig.json [hunk 4]
```diff
@@ -5,8 +5,12 @@
     "importHelpers": true,
     "lib": ["es2018"],
     "module": "commonjs",
+    "noImplicitAny": false,
```
Comment (tiagonapoli) [inline tsconfig.json:8]:
```suggestion
    "noImplicitAny": true,
```

File: package.json [hunk 5]
```diff
@@ -13,18 +13,19 @@
...
     "jest": "^25.1.0",
-    "nodemon-notifier-cli": "https://github.com/Slessi/nodemon-notifier-cli.git",
     "nodemon": "^2.0.2",
+    "nodemon-notifier-cli": "https://github.com/Slessi/nodemon-notifier-cli.git",
     "prettier": "^2.0.1",
     "ts-jest": "^25.2.1",
     "ts-node": "^8",
-    "typescript": "^3.8.2"
+    "typescript": "^3.8.2",
+    "vtex": "^2.103.1-beta"
```
Comment (tiagonapoli) [inline package.json:28]: When we release 'vtex' with the new changes we have to remember to update here

File: package.json [hunk 6]
```diff
@@ -13,18 +13,19 @@
     "@oclif/plugin-help": "^3",
     "@types/jest": "^26.0.3",
     "@types/node": "^14.0.14",
-    "eslint-config-oclif-typescript": "^0.1",
+    "eslint": "^6.8.0",
     "eslint-config-oclif": "^3.1",
+    "eslint-config-oclif-typescript": "^0.1",
```
Comment (tiagonapoli) [inline package.json:18]: No need for these - mirror the eslint configs used on toolbelt, which is only:
```
 "eslint-config-vtex": "^12.3.1",
```

File: .gitignore (hunk 3 above)
Comment (VerasThiago) [reply .gitignore:13]: This file is already there. 

https://github.com/vtex/cli-plugin-template/blob/f1b8ccf5cb5f9f677046b9c5b18c3ae1aa89e696/.gitignore#L2

File: package.json (hunk 5 above)
Comment (VerasThiago) [reply package.json:28]: Sure

File: .gitignore (hunk 3 above)
Comment (tiagonapoli) [reply .gitignore:13]: Ops, sorry, didn't saw

**Pull request merged**